apirest.md

```diff
@@ -131,20 +132,24 @@ $ curl -X GET \
 
 < 200 OK
 < {
-   "session_token": "83af7e620c83a50a18d3eac2f6ed05a3ca0bea62",
-   "users_id": "42"
```

Comment:
I guess users_id should be kind of "deprecated" instead of being removed (to keep backward compatibility)

Reply:
`users_id` is a 9.5 feature.

Reply:
Oh, ok :)